es-vue-base/nuxt/index.js

```diff
@@ -0,0 +1,27 @@
+import { join } from 'path';
+
+export default function autoExport(customOptions) {
+    const options = {
+        components: true,
+        icons: true,
+        ...customOptions,
+    };
+
+    this.nuxt.hook('components:dirs', (dirs) => {
+        // Auto import components
+        if (options.components) {
+            dirs.push({
+                path: join(__dirname, '../src/lib-components'),
+                extensions: ['vue'],
+            });
+        }
+
+        // Auto import icons
```

Comment:
Some components include icons, would they break if not auto-importing icons? Should we just auto-import all components?

Reply:
They will not break. The auto importing happens in side nuxt not vue so those icons used by components in `es-vue-base` keep their direct import statement.